src/target/riscv/riscv.h

```diff
@@ -111,6 +111,13 @@ typedef struct {
 	char *name;
 } range_list_t;
 
+struct reg_name;
+
+struct reg_name_table {
+	unsigned int names_num;
```

Comment:
For clarity, this could be called e.g.:

```suggestion
	unsigned int num_entries;
```

Reply:
Addressed.